lua/lspconfig/vdmj.lua

```diff
@@ -0,0 +1,106 @@
+local configs = require 'lspconfig/configs'
+local util = require 'lspconfig/util'
+
+local server_name = 'vdmj'
+
+local mavenrepo = util.path.join(tostring(vim.fn.getenv 'HOME'), '/.m2/repository/com/fujitsu')
```

Comment:
Use `vim.env` instead of `vim.fn.getenv`.

```suggestion
local mavenrepo = util.path.join(vim.env.HOME, '/.m2/repository/com/fujitsu')
```